model_analyzer/config/generate/model_config_generator.py

```diff
@@ -39,6 +39,7 @@ def __init__(self, config, model, client):
         self._base_model = model
         self._base_model_name = model.model_name()
         self._model_name_index = 0
+        self._default_param_combo = {}
```

Comment:
this feels like it should be a constant? What is the correct way to do that in python?

Reply:
If the constant is only used in this file, you can pull it out to a class variable like here: https://github.com/triton-inference-server/model_analyzer/blob/7a6792e94969fdddbec8ee5a62b2b80a2b9e91ad/model_analyzer/reports/report_factory.py#L29

If you are using this in multiple files, you can add it to the constants file and import it.